src/mlpack/tests/akfn_test.cpp

```diff
@@ -163,6 +163,12 @@ TEST_CASE("AKFNSingleCoverTreeTest", "[AKFNTest]")
  *
  * Errors are produced if the results are not according to relative error.
  */
+
+// These macros are added in order to stop executing these tests on aarch64.
+// These is unknown issue with Dual Cover tree test on these architecture
+// The issue is explained in #2869 on github.
+// Once the issue is resolved. These macros can be removed safely.
```

Comment:
Would go with the singular form here, `[...] resolved, the macro can be removed [...]`.